samcli/commands/list/json_consumer.py

```diff
@@ -0,0 +1,10 @@
+"""
+The json consumer for 'sam list'
+"""
+import click
+from samcli.lib.list.list_interfaces import ListInfoPullerConsumer
+
+
+class JsonConsumer(ListInfoPullerConsumer):
```

Comment:
It says JSON consumer but it is accepting `str`, should this be re-named to `StringConsumer`?

Reply:
It's called json consumer because it outputs the data in json format. Maybe I could write a docstring to clarify that?

Reply:
I see but from implementation point of view, it prints whatever string that is been given to it. So I would rather name it as string consumer.

Reply:
Sure, but can I leave some sort of indication of the json output in the name? Since there will be a consumer that outputs a table format

Reply:
Sure, I would suggest to put some documentation in it.

Reply:
Changed it and added doc